build.gradle

```diff
@@ -85,6 +95,11 @@ license {
   excludes(["**/*.txt","**/gradlew.*","**/gradle-wrapper.*","src/test/resources/HelloDeployment/.gradle/**"])
 }
 
+clean {
```

Comment:
Shouldn't this be part of clean.dolast?

Reply:
Yes this should have been part of a clean.doLast {} block.  I'll push the update to my pull request.